test/rubygems/test_project_sanity.rb

```diff
@@ -12,4 +12,12 @@ def test_manifest_is_up_to_date
     assert status.success?, "Expected Manifest.txt to be up to date, but it's not. Run `rake update_manifest` to sync it."
   end
 
+  def test_require_rubygems_package
+    require "rubygems/test_case"
```

Comment:
This require needs to go at the top of the file, right before inheriting from `Gem::TestCase`.

Reply:
good catch. Did it before changing the parent class, why is this working?

Reply:
Because `rubygems/test_case` is being required by other tests. If you run the test file in isolation (`rake TEST=test/rubygems/test_project_sanity.tb`) it will fail though.

Reply:
done